src/formElementPlugins/number/src/ApplicationView.tsx

```diff
@@ -162,18 +196,32 @@ const parseInput = (
   textInput: string | null | undefined,
   numberFormatter: Intl.NumberFormat,
   simple: boolean,
-  prefix: string,
-  suffix: string
-): [number | null, string | null] => {
-  if (!textInput) return [null, null]
+  prefix: string | undefined,
+  suffix: string | undefined,
+  suffixPlural: string | undefined
+): {
+  number: number | null
+  formattedNumber: string | null
+  currentSuffix: string | undefined
+  fullText: string | null
+} => {
+  if (!textInput)
+    return { number: null, formattedNumber: null, currentSuffix: suffixPlural, fullText: null }
   // Strip out any text that isn't a digit, "-" or "." to create the number
   const rawDigits = textInput?.replace(/[^\d\.\-]/g, '')
-  if (rawDigits === '') return [null, textInput]
+  if (rawDigits === '')
+    return { number: null, formattedNumber: textInput, currentSuffix: suffixPlural, fullText: null }
   const number: number = Number(rawDigits)
-  const text: string = simple
-    ? String(number)
-    : ((prefix + numberFormatter.format(number) + suffix) as string)
-  return [number, text]
+  const prefixString = prefix ? prefix + ' ' : ''
+  const suffixString = suffix ? ' ' + (number === 1 ? suffix : suffixPlural) : ''
```

Comment:
Pad prefix and suffix with a space for concatenating with formatted number value.